crates/jql/Cargo.toml

```diff
@@ -16,8 +16,8 @@ anyhow = "1.0.70"
 clap = { version = "4.2.4", features = ["derive"] }
 colored_json = { version = "3.1.0" }
 jql-runner = { path = "../jql-runner", version = "6.0.6" }
-serde = "1.0.160"
-serde_stacker = "0.1.8"
+mimalloc = "0.1.36"
 serde_json = { workspace = true }
+simd-json = "0.9.0"
```

Comment:
Didn't see it mentioned in the PR; how much did this improve performance?

Reply:
That's the missing piece, not tested yet :)

Reply:
I made a small hyperfine shell script comparing the current binary from main and a release build from this branch.

There's definitely some improvement. Ideally this should be tested against a bway igger JSON file.

```sh
Benchmark 1: cat /home/yamafaktory/dev/jql/assets/github-repositories.json | jql '|>{"name","full_name"}' > /dev/null
  Time (mean ± σ):     597.2 ms ±  12.1 ms    [User: 880.7 ms, System: 165.1 ms]
  Range (min … max):   569.9 ms … 629.4 ms    100 runs
 
Benchmark 2: cat /home/yamafaktory/dev/jql/assets/github-repositories.json | /home/yamafaktory/dev/jql/target/release/jql '|>{"name","full_name"}' > /de
v/null
  Time (mean ± σ):     555.8 ms ±  13.9 ms    [User: 1001.4 ms, System: 130.3 ms]
  Range (min … max):   531.7 ms … 648.2 ms    100 runs
 
Summary
  'cat /home/yamafaktory/dev/jql/assets/github-repositories.json | /home/yamafaktory/dev/jql/target/release/jql '|>{"name","full_name"}' > /dev/null' ra
n
    1.07 ± 0.03 times faster than 'cat /home/yamafaktory/dev/jql/assets/github-repositories.json | jql '|>{"name","full_name"}' > /dev/null'
```